app/models/user.rb

```diff
@@ -147,7 +147,7 @@ class MissingFrilansFinansIdError < RuntimeError; end
   translates :description, :job_experience, :education, :competence_text
 
   # NOTE: This is necessary for nested activeadmin has_many form
-  accepts_nested_attributes_for :user_skills, :user_languages, :user_interests
+  accepts_nested_attributes_for :user_skills, :user_languages, :user_interests, :user_documents # rubocop:disable Metrics/LineLength
```

Comment:
This is needed to allow `user_document`s being created [here](https#diff-eae04998dbc8a9a55c9dc9db53fc1eb1L375)